src/shared/user-helpers.ts

```diff
@@ -1,13 +1,58 @@
 import { UserProfile } from "@auth0/nextjs-auth0";
+import md5 from "md5";
+import querystring from "query-string";
```

Comment:
I guess we can use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams instead. Saves us [a few KB](https://bundlephobia.com/package/query-string@7.1.0).

The popularity of `query-string` [is declining](https://www.npmtrends.com/query-string), which can be explained by its replacement with a native API.

<img width="1326" alt="Screenshot 2022-02-01 at 09 50 53" src="https://user-images.githubusercontent.com/608862/151946981-95941532-843a-4494-bf93-6881be61b486.png">

If we choose to use some off-the-shelf gravatar library, we won’t have to bother about it.